M2/GNUmakefile.in

```diff
@@ -210,15 +210,7 @@ ifeq (@BUILD_givaro@,yes)
 submodules/fflas_ffpack/Makefile: install-in-givaro
 endif
 
-all-in-Macaulay2:install-licenses
-
-install-licenses:all-in-submodules
```

Comment:
We still need to install the licenses, so I think this code has to stay.

Reply:
Oh, I see you moved it to another makefile.  Is there a reason to do so?

Reply:
Seems like `install-programs` and `install-licenses` should go together, especially given that we typically install licenses only for the files that we distribute, which are basically all programs because most libraries are statically linked and hence not distributed.